lib/elixir_conf_africa_web/live/home_live/index.ex

```diff
@@ -2,23 +2,45 @@ defmodule ElixirConfAfricaWeb.HomeLive.Index do
   use ElixirConfAfricaWeb, :live_view
   alias ElixirConfAfrica.Events
 
+  alias ElixirConfAfrica.Cart
+
   def mount(_params, _session, socket) do
-    # these value are more static and we should find away of display this data to home page
-    event_name = "ElixirConf Africa #{get_current_year()}"
+    event_name = "ElixirConf Africa #{get_upcoming_year()}"
 
     event =
       Events.get_event_with_ticket_types_by_event_name(event_name)
 
-    available_ticket = Events.get_total_number_of_available_tickets(event_name)
+    available_tickets = Events.get_total_number_of_available_tickets(event_name)
 
     {:ok,
      socket
-     |> assign(:event, event)}
-    |> assign(available_ticket: available_ticket)
+     |> assign(:event, event)
+     |> assign(:cart, [])
+     |> assign(available_tickets: available_tickets)}
   end
 
-  defp get_current_year do
+  defp get_upcoming_year do
     %{year: year} = DateTime.utc_now()
-    year
+    year + 1
+  end
+
+  def handle_event("add_to_cart", %{"id" => id}, socket) do
+    cart_item_ids = Cart.cart_list_ids(socket.assigns.cart)
+
+    if Enum.member?(cart_item_ids, String.to_integer(id)) do
```

Comment:
whats happening here? It seems whether this is true or false the item is still added to the cart and only the flash message is different

Reply:
How @okothkongo  suggested we handle this is such that , if there is an item in the cart , and someone adds the same item to the cart , it updates the quantity of that item in the cart and updates the user that the quantity has been added as opposed to disabling the add to cart for an item already in the cart